caps/forms.py

```diff
@@ -1,14 +1,46 @@
-from django.forms import CharField, BooleanField, ModelChoiceField, Select
+"""
+Forms for the CAPS app
+"""
+
+
+from django.forms import CharField, ChoiceField, ModelChoiceField, RadioSelect, Select
 from haystack.forms import SearchForm
-from haystack.query import SearchQuerySet
 from haystack.inputs import Exact
+from haystack.query import SearchQuerySet
 
-from caps.models import Distance, ComparisonType, Council
+from caps.models import ComparisonType, Council, Distance
+from caps.search_funcs import (
+    condense_highlights,
+    get_semantic_query,
+    highlighter_config,
+    fuller_highlighter_config,
+)
+from django.conf import settings
 
 
 class HighlightedSearchForm(SearchForm):
+    """
+    Search form for Haystack/Solr searches of the documents
+    """
+
     council_name = CharField(required=False)
-    exact = BooleanField(required=False)
+
+    MATCH_NORMAL = "normal"
+    MATCH_EXACT = "exact"
+    MATCH_RELATED = "related"
+    MATCH_RELATED_LOOSE = "related_loose"
+
+    MATCH_CHOICES = (
```

Comment:
> Is the way the new search options are described intuitive enough? What exact descriptions are required?

My instinct is, if we think our semantic search is useful, we should make it the default.

But regardless of whether we make it the default or not, we should present the "Search method" options in order of increasing or decreasing specificity, eg:

```
    MATCH_CHOICES = (
        (MATCH_RELATED_LOOSE, "Include as many terms as possible"),
        (MATCH_RELATED, "Include closely related terms (default)"),
        (MATCH_NORMAL, "Search for any of the provided words"),
        (MATCH_EXACT, "Search for the provided phrase exactly"),
    )
```

And the second option would be selected by default.